sync/src/block_connector/metrics.rs

```diff
@@ -56,12 +56,12 @@ impl ChainMetrics {
             registry,
         )?;
 
-        let chain_block_num = register(
+        let chain_block = register(
             UIntGauge::with_opts(Opts::new("chain_block_num", "how many block in main chain"))?,
             registry,
         )?;
 
-        let chain_txn_num = register(
+        let chain_txn = register(
```

Comment:
it confused pub struct SignedUserTransaction and pub struct RawUserTransaction, you can search this code.
There are some naming is related with our project, these struct i mentioned, you can see it. English is not my first language so please excuse any mistakes